internal/server/recon/find_entities.go

```diff
@@ -158,7 +147,159 @@ func BulkFindEntities(
 	return resp, nil
 }
 
-func resolvePlaceIDs(
+func resolveDCIDs(
+	ctx context.Context,
+	mapsClient *maps.Client,
+	store *store.Store,
+	entityInfoSet map[entityInfo]struct{},
+) (
+	map[entityInfo][]string, /* entityInfo -> [DCID] */
+	map[string]struct{}, /* [DCID] for all entities */
+	error,
+) {
+	// First try to resolve DCIDs by RecognizePlaces.
+	entityInfoToDCIDSet, dcidSet, err := resolveWithRecognizePlaces(
+		ctx, store, entityInfoSet)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// See if there are any entities that cannot be resolved by RecognizePlaces.
```

Comment:
1. I think it might be useful to have set some debug info so we can tell how we matched?  This will be useful in evaluating/debugging the results?  Separate PR is fine.

2. Also, I'm wondering about doing calls parallel-y, and if both have results, prefer recognize-places, but use Maps API as signal to reorder the results.  Could add a TODO maybe?

Reply:
Added TODOs